cfgov/unprocessed/css/apps/owning-a-home/brand-footer.less

```diff
@@ -0,0 +1,62 @@
+.brand-footer {
+    background-color: @gray-5;
+    padding-top: @grid_gutter-width;
+    padding-bottom: 60px;
+    border-top: 1px solid @gray-40;
+
+    .title-link {
+        border-bottom-width: 1px;
+    }
+
+    &_img {
+        text-align: center;
+
+        .respond-to-min( @bp-sm-min, {
+            text-align: left;
+        } );
+
+        a:after {
+            display: none !important;
+        }
+
+        img {
+            width: 150px;
+            padding-bottom: unit( ( @grid_gutter-width - 5 ) / @base-font-size-px, em );
+        }
+    }
+
+    &_headline {
+        .grid_column( 12 );
+
+        .cf-icon {
+            position: relative;
+            top: 2px;
+            padding-right: 2px;
+            color: @green;
+            font-size: 1.25em;
+        }
+    }
+
+    &_container {
+        padding-top: unit( ( @grid_gutter-width * 1.5) / @base-font-size-px, em );
+
+        &-1 {
+            padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+
+        .grid_column( 12 );
+
+        .respond-to-min( @bp-sm-min, {
+            .grid_column( 6 );
+
+            &-2 {
+                padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+            }
+        } );
+
+        .respond-to-min( 1000px, {
```

Comment:
Should `1000px` be a breakpoint variable?

Reply:
I left this as-is because it is a one-off.